raiwidgets/raiwidgets/error_analysis_dashboard.py

```diff
@@ -95,35 +95,6 @@ class ErrorAnalysisDashboard(Dashboard):
         Specify less than 10k points for optimal performance.
     :type sample_dataset: pd.DataFrame or numpy.ndarray or list[][]
 
-    :Example:
-
-    Run simple view of error analysis with just predictions and true labels
```

Comment:
why are these examples being removed?  they are shown in the docs page
https://erroranalysis.ai/raiwidgets.html
![image](https://user-images.githubusercontent.com/24683184/150462302-4d1f5cba-d5eb-42d7-b5b4-3a6c5bca94f5.png)

I think these are good examples to have?